Composer/packages/server/src/models/bot/__tests__/botProject.test.ts

```diff
@@ -291,17 +291,31 @@ describe('skill operations', () => {
   xit('should create skill files', async () => {
     await proj.init();
 
-    const url = 'https://luhantest0625.azurewebsites.net/manifests/Empty_45-2-1-manifest.json';
+    const url = 'https://xxx/manifests/Empty_45-2-1-manifest.json';
```

Comment:
first, these tests are disabled, you should enable them and see if they pass.
second,. move this huge mocked zipContent outside each test and re-use it instead of re-defining it every time (assuming they are the same).

Reply:
move the variable outside